collect_app/src/main/java/org/odk/collect/android/views/MediaLayout.java

```diff
@@ -232,32 +232,41 @@ public void onClick(View v) {
                         imageView.setImageBitmap(b);
                         imageView.setId(imageId);
 
-                        if (bigImageURI != null) {
-                            imageView.setOnClickListener(new OnClickListener() {
-                                String bigImageFilename = ReferenceManager.instance()
-                                        .DeriveReference(bigImageURI).getLocalURI();
-                                File bigImage = new File(bigImageFilename);
-
+                        imageView.setOnClickListener(new OnClickListener() {
+                            @Override
+                            public void onClick(View v) {
+                                Collect.getInstance().getActivityLogger().logInstanceAction(
```

Comment:
I know we really need to deprecate this but in the mean time, to preserve semantics, I think it should go in the `bigImageURI != null` block.